services/src/main/java/org/exoplatform/task/util/CommentUtil.java

```diff
@@ -40,6 +49,7 @@ public static String formatMention(String text, UserService userService) {
     }
     HTMLEntityEncoder encoder = HTMLEntityEncoder.getInstance();
     StringBuilder sb = new StringBuilder();
+    String lang = getCurrentUserLanguage(ConversationState.getCurrent().getIdentity().getUserId());
```

Comment:
If this is called by a job, you will not have a user.
It can be called by MailTemplateProvider, so it is possible to not have a user.

I think it is better to make the lang as parameter of the function.
If you dont want to change everywhere in the code, you can add a function formatMention, without the parameter lang, which call the formatMention with the parameter lang, with default value.

But I think the solution to change everywhere is better